cli/app.go

```diff
@@ -926,7 +926,7 @@ var app = &cli.App{
 									UsageText: createUsageText("train submit custom from-registry",
 										[]string{
 											datasetFlagDatasetID, generalFlagOrgID, trainFlagModelName,
-											mlTrainingFlagName, mlTrainingFlagVersion,
+											mlTrainingFlagName, mlTrainingFlagVersion, trainCliArgs,
```

Comment:
Can we make this mlTraining so it follows the same convention as all the other flags?